js/quote_tool.js

```diff
@@ -1 +1,64 @@
-// Qouting tool logic should be placed here.
\ No newline at end of file
+// Qouting tool logic should be placed here.
+
+// Remember to import consts.js if you will use this file 
+
+/* calculateCost calculates the value of the package that the user prefers 
+ * by giving a value of each one of the options selected by the user
+ * 
+ * The expected objet structure is :
+ * {
+ *  numberOfPeople: integer,
+ *  numberOfNigth: integer,
+ *  trasnport: object, 
+ *  tour: object,
+ *  hike: object,
+ * } 
+ * 
+ * trasnport: {
+ *  medNec: boolean,
+ *  necCap: boolean,
+ *  capNec: boolean,
+ *  necMed: boolean,
+ * }
+ * tour: {
+ *  aguacatePlayasoledad: boolean,
+ *  sapzurroLamiel: boolean,
+ * }
+ * hike: {
+ *  elCieloEltrebol: boolean,
+ *  elParaiso: boolean,
+ *  laCoquerita: object,
+ * }
+ * laCoquerita: {
+ *  hike: boolean,
+ *  guide: boolean,
+ * }
+ */
```

Comment:
Amazing that you've documented this code but I recommend you read how to use the jsdoc for the more formal situations.

Reference: http://usejsdoc.org/about-getting-started.html